### PR TITLE
V1.14.0 changelog

### PR DESCRIPTION
**Pull request by tsroten**
## Description

Bumps the version in the changelog and puts the Features section at the top.

Also, adds the Python 2 trove classifier, which was missing from `setup.py`.

### CODE REVIEW

**Comment by tsroten** [comment]
@dbcli/mycli-core It's time for another release! 🍾 🎆 If you have time, please review the changes 😄

**Comment by j-bennet** [comment]
Looks good! 👍

File: changelog.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-
-* Fix source command for script in current working directory. (Thanks: [Dick Marinus]).
-
 Features:
 ---------
 
 * Add `watch [seconds] query` command to repeat a query every [seconds] seconds (by default 5). (Thanks: [David Caro](https://github.com/Terseus))
 * Default to unix socket connection if host and port are unspecified. This simplifies authentication on some systems and matches mysql behaviour.
-* Add support for positional parameters to favorite queries
+* Add support for positional parameters to favorite queries.
```
Comment (amjith) [inline changelog.md]: Who do we thank for this change?

Comment (j-bennet) [reply changelog.md]: @scrappysoft - I guess he didn't want thanks: https://github.com/dbcli/mycli/pull/542.

**Pull request merged**